spotifycli/test/test_spotifycli.py

```diff
@@ -0,0 +1,163 @@
+
+    def test_cli_status_position(self):
+        with patch('sys.argv', ["spotifycli", "--statusposition"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            output = buffer.getvalue()
+            self.assertIn('00:01', output)
+            self.assertIn('00:10', output)
+
+    def test_cli_status_short(self):
+        with patch('sys.argv', ["spotifycli", "--statusshort"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            output = buffer.getvalue()
+            self.assertIn('Beatles', output)
+            self.assertIn('Yesterday', output)
+
+    def test_cli_song(self):
+        with patch('sys.argv', ["spotifycli", "--song"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            self.assertIn("Yesterday", buffer.getvalue())
+
+    def test_cli_song_short(self):
+        # TODO: Change to a long song title to check the trimming
+        with patch('sys.argv', ["spotifycli", "--songshort"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            self.assertIn("Yesterday", buffer.getvalue())
+
+    def test_cli_artist(self):
+        with patch('sys.argv', ["spotifycli", "--artist"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            self.assertIn("Beatles", buffer.getvalue())
+
+    def test_cli_artist_short(self):
+        # TODO: Change to a long artist name to check the trimming
+        with patch('sys.argv', ["spotifycli", "--artistshort"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            self.assertIn("Beatles", buffer.getvalue())
+
+    def test_cli_album(self):
+        with patch('sys.argv', ["spotifycli", "--album"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            self.assertIn("Help!", buffer.getvalue())
+
+    def test_cli_position(self):
+        with patch('sys.argv', ["spotifycli", "--position"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            output = buffer.getvalue()
+            self.assertIn('00:01', output)
+            self.assertIn('00:10', output)
+
+    def test_cli_art_url(self):
+        with patch('sys.argv', ["spotifycli", "--arturl"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            self.assertIn('http', buffer.getvalue())
+
+    def test_cli_playback_status(self):
+        with patch('sys.argv', ["spotifycli", "--playbackstatus"]), patch('sys.stdout', new_callable=StringIO) as buffer:
+            main()
+            playback_symbols = ['▶', '▮▮', '■']
+            output = buffer.getvalue()
+            self.assertTrue(any([x in output for x in playback_symbols]))
+
+    @skip('Lyrics dont seem to be working right now...is it still valid?')
+    def test_cli_lyrics(self):
+        with patch('sys.argv', ["spotifycli", "--lyrics"]):
+            main()
```

Comment:
The following tests are decorated with `@skip` because they currently fail.

For this one, I'm not convinced the lyrics database is still available -- maybe I'm missing something.  But if not, that command line switch should probably just be hidden away for now.